Formula/file-roller.rb

```diff
@@ -11,30 +11,48 @@ class FileRoller < Formula
     sha256 "05ca1ccdd5771e574bf4b1bad58fa0ddc6d2db65d2fda7999912ced5e0cb704e" => :mavericks
   end
 
-  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
```

Comment:
I thought that was self evident since there's no head build but happy to add a comment. ;)